robotpy_build/tool.py

```diff
@@ -256,6 +261,180 @@ def run(self, args):
         pprint.pprint(platforms.get_platform_override_keys(p))
 
 
+class MavenParser:
+
+    os_names = ["linux", "windows", "osx"]
+    known_arch_names = [
```

Comment:
Can we pull this from platforms.py instead?

Reply:
I've changed it to use known os and archs from platforms.py. This only applies when the brute force flag is used. Regularly, with html scraping, it just finds all available platforms in the html. As far as I know, the only difference between the two modes is detection of aarch64bionic, raspbian, and jetsontx.